azuredevops/internal/acceptancetests/resource_check_business_hours_test.go

```diff
@@ -78,6 +78,7 @@ func TestAccCheckBusinessHours_complete(t *testing.T) {
 					resource.TestCheckResourceAttr(tfCheckNode, "friday", "false"),
 					resource.TestCheckResourceAttr(tfCheckNode, "saturday", "false"),
 					resource.TestCheckResourceAttr(tfCheckNode, "sunday", "false"),
+					resource.TestCheckResourceAttr(tfCheckNode, "timeout", "50000"),
```

Comment:
default is `1440` not `5000`

Reply:
What about the values that are used in the Acceptance tests for "Check Approval"?
See:
https://github.com/microsoft/terraform-provider-azuredevops/blob/bd2d69d460e62ea22f0d11252d58268bd5c16130/azuredevops/internal/acceptancetests/resource_check_approval_test.go#L33

https://github.com/microsoft/terraform-provider-azuredevops/blob/bd2d69d460e62ea22f0d11252d58268bd5c16130/azuredevops/internal/acceptancetests/resource_check_approval_test.go#L59